packages/react-ux-capture-example/src/Foo.jsx

```diff
@@ -1,11 +1,33 @@
 import React from 'react';
-import Inline from './marks/Inline';
+
+import UXCaptureInlineMark from '@meetup/react-ux-capture/lib/UXCaptureInlineMark';
+import UXCaptureImageLoad from '@meetup/react-ux-capture/lib/UXCaptureImageLoad';
+import UXCaptureInteractiveMark from '@meetup/react-ux-capture/lib/UXCaptureInteractiveMark';
+
 import Page from './Page';
+import { getBoxStyle } from './ZoneHelper';
+import Lazy from './Lazy';
 
-const destinationVerified = ['ux-text-foo-title', 'ux-image-onload-script-logo'];
+const destinationVerified = [
+	'ux-text-foo-title',
+	'ux-image-inline-logo',
+	'ux-image-onload-logo',
+];
 const primaryContentDisplayed = ['ux-text-foo-primary'];
-const secondaryContentDisplayed = ['ux-text-foo-secondary'];
-const primaryActionAvailable = ['ux-button-foo-primary'];
+const secondaryContentDisplayed = [
+	'ux-text-foo-secondary',
```

Comment:
Another example of I'd move away from using `foo` and make it real content with a descriptive mark name, as you do for the kitten image.